tests/LSP6KeyManager/internals/AllowedERC725YKeys.internal.ts

```diff
@@ -1,168 +1,728 @@
+        "0xfd63b8f031e1f4c43fbb4956e08c686aa350a051d4d3e77a1e1c7f70366207b2",
+        "0xf4fafa18dc0e9916a17a5d14b39a4dc68fb56cfaf964aceca8456b424331adb4",
+        "0x92f9ce0fd87f5477d0aec48786bfebf80a51700879505dea17852f0777b114d3",
+        "0x0c02c965d192a6666110f3806dc2cd489ce2f7ca43f08a9ff33ad74d2fb30dc7",
+        "0x108eae0761cc2f55fab5acf94bc557dd0e550e60be19de83c513cd17821227a8",
+        "0xe2bfbedc99949767be4c79d5d10deaffc1c207560d29e4070f450faabb562d3b",
+        "0xe5f29667509d955384331154d9fa3b9e53ae230c44e722983a42f4d9608ef9b6",
+        "0x9fedfe7c7366c70c52b6f11531196d0c5baa77abd16117ae30dc4eeb16dd6da2",
+        "0x8741530fb57ca8556c5e7b45ffac62c178d8c0ff070ff1c99652e3f099997fa6",
+      ];
+      compactBytesArray_2d =
+        dataKeys.firstDynamicKey.length +
+        dataKeys.firstDynamicKey.key.toString().substring(2) +
+        dataKeys.secondDynamicKey.length.toString().substring(2) +
+        dataKeys.secondDynamicKey.key.toString().substring(2);
+      compactBytesArray_2f =
+        dataKeys.firstFixedKey.length +
+        dataKeys.firstFixedKey.key.toString().substring(2) +
+        dataKeys.secondFixedKey.length.toString().substring(2) +
+        dataKeys.secondFixedKey.key.toString().substring(2);
+      compactBytesArray_2d_2f =
+        dataKeys.firstDynamicKey.length +
+        dataKeys.firstDynamicKey.key.toString().substring(2) +
+        dataKeys.secondDynamicKey.length.toString().substring(2) +
+        dataKeys.secondDynamicKey.key.toString().substring(2) +
+        dataKeys.firstFixedKey.length.toString().substring(2) +
+        dataKeys.firstFixedKey.key.toString().substring(2) +
+        dataKeys.secondFixedKey.length.toString().substring(2) +
+        dataKeys.secondFixedKey.key.toString().substring(2);
+      compactBytesArray_mixed_d_f =
+        dataKeys.firstDynamicKey.length +
+        dataKeys.firstDynamicKey.key.toString().substring(2) +
+        dataKeys.firstFixedKey.length.toString().substring(2) +
+        dataKeys.firstFixedKey.key.toString().substring(2) +
+        dataKeys.secondDynamicKey.length.toString().substring(2) +
+        dataKeys.secondDynamicKey.key.toString().substring(2) +
+        dataKeys.thirdDynamicKey.length.toString().substring(2) +
+        dataKeys.thirdDynamicKey.key.toString().substring(2) +
+        dataKeys.secondFixedKey.length.toString().substring(2) +
+        dataKeys.secondFixedKey.key.toString().substring(2) +
+        dataKeys.fourthDynamicKey.length.toString().substring(2) +
+        dataKeys.fourthDynamicKey.key.toString().substring(2) +
+        dataKeys.thirdFixedKey.length.toString().substring(2) +
+        dataKeys.thirdFixedKey.key.toString().substring(2);
+    });
 
-  describe("keyType: Singleton", () => {
-    let controllerCanSetOneKey: SignerWithAddress;
+    describe("`isCompactBytesArray(..)`", () => {
+      it("Verifying the validity of a CompactBytesArray containing 2 dynamic keys", async () => {
```

Comment:
Please explain what it should return (`true` or `false`).

```suggestion
      it("should return true for a CompactBytesArray containing 2 dynamic keys", async () => {
```